src/rendertron.ts

```diff
@@ -56,6 +60,7 @@ export class Rendertron {
       response: express.Response) {
     if (!this.renderer) {
       console.error(`No renderer yet`);
```

Comment:
Can this actually happen? It looks like you could never have started listening unless you had initialized renderer. So maybe just throw?

Reply:
Done.